src/flotilla/scheduler/doctor.py

```diff
@@ -3,12 +3,10 @@
 
 logger = logging.getLogger('flotilla')
 
-SERVICE_EXPIRY = 10
-
 
 class ServiceDoctor(object):
     def __init__(self, db, elb):
-        self._db = db
+        self.db = db
```

Comment:
This `db` is not private to this class? http://stackoverflow.com/a/1301369/1222881